### ISSUE TITLE
Running action causes `Error: Invalid context`

### ISSUE THREAD

**Issue by daffl** [technote-space/create-pr-action#137]
## Describe the bug: バグの概要

Using this GitHub action with the configuration at https://github.com/feathersjs/feathers/blob/412783455dc854c4237a3b6463458eab63cd1b10/.github/workflows/update-dependencies.yml causes the following error:

```
Error: Invalid context.
    at /home/runner/work/_actions/technote-space/create-pr-action/v1/node_modules/@technote-space/github-action-pr-helper/dist/utils/variables.js:48:19
    at Generator.next (<anonymous>)
    at /home/runner/work/_actions/technote-space/create-pr-action/v1/node_modules/@technote-space/github-action-pr-helper/dist/utils/variables.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/home/runner/work/_actions/technote-space/create-pr-action/v1/node_modules/@technote-space/github-action-pr-helper/dist/utils/variables.js:4:12)
    at Object.replace (/home/runner/work/_actions/technote-space/create-pr-action/v1/node_modules/@technote-space/github-action-pr-helper/dist/utils/variables.js:46:49)
    at /home/runner/work/_actions/technote-space/create-pr-action/v1/node_modules/@technote-space/github-action-helper/dist/utils.js:144:95
    at Generator.next (<anonymous>)
    at /home/runner/work/_actions/technote-space/create-pr-action/v1/node_modules/@technote-space/github-action-helper/dist/utils.js:8:71
    at new Promise (<anonymous>)
```

## To Reproduce: 再現手順

You can view the full log at https://github.com/feathersjs/feathers/runs/524687311

## Operating environment: バグが発生した環境

GitHub actions

**Comment by technote-space** [comment]
Thank you for your bug report.
I will fix this soon.

**Comment by technote-space** [comment]
technote-space/github-action-pr-helper#172

**Comment by technote-space** [comment]
I have released a version that fixes this problem (v1.4.2).

Unless you set an option `ONLY_DEFAULT_BRANCH` to  true , v1 will cover all PRs.
If you want to update and create a PR only with the default branch, set `ONLY_DEFAULT_BRANCH` to true or use v2
